Packs/mnemonicMDR/Integrations/ArgusManagedDefence/ArgusManagedDefence.yml

```diff
@@ -1,84 +1,107 @@
-category: Data Enrichment & Threat Intelligence
 commonfields:
   id: ArgusManagedDefence
   version: -1
+name: ArgusManagedDefence
+display: mnemonic MDR - Argus Managed Defence (Partner Contribution)
+category: Data Enrichment & Threat Intelligence
+description: Rapidly detect, analyse and respond to security threats with mnemonic’s
+  leading Managed Detection and Response (MDR) service.
 configuration:
 - display: Fetch incidents
   name: isFetch
-  required: false
   type: 8
+  required: false
 - display: Incident type
   name: incidentType
-  required: false
   type: 13
-- additionalinfo: 0 means up to system limit (100 000)
-  defaultvalue: '25'
-  display: Maximum number of incidents per fetch
-  name: max_fetch
   required: false
+- display: API URL
+  name: api_url
+  defaultvalue: https://api.mnemonic.no
   type: 0
+  required: true
+  additionalinfo: "URL to Argus' API Endpoint."
 - display: API Key
   name: api_key
-  required: true
   type: 4
-- defaultvalue: low
-  display: Minimum severity of alerts to fetch
-  name: min_severity
-  options:
-  - low
-  - medium
-  - high
-  - critical
   required: true
+  additionalinfo: "API Key of API user in Argus."
+- display: Minimum severity of alerts to fetch
+  name: min_severity
+  defaultvalue: Low
   type: 15
-- defaultvalue: 3 days
-  display: First fetch time
+  required: true
+  options:
+  - Low
+  - Medium
+  - High
+  - Critical
```

Comment:
Is there a reason to change it to uppercase?
It will break BC, and the code is still using the lowercase values.

Reply:
It looks better in GUI. I believe if someone already has configured it to say e.g. "low", this would not affect this update as it runs lower() on it anyway. 
If, however, someone manage to give this new default value to a non-updated instance, it would break. But an old instance would still show lower-case values?